pkg/micropython/doc.txt

```diff
@@ -0,0 +1,41 @@
+/**
```

Comment:
Since the RIOT port seems to have several RIOT-specific additions, some more documentation would be welcome.

Reply:
Please help out!

Reply:
No, man, this is _your_ feature.